src/clr-addons/themes/phs/phs-theme.scss

```diff
@@ -1,3 +1,5 @@
+$clr-color-hue: 183.2;
```

Comment:
actually, shouldnt we move this into the variables.color file of the phs folder - this is just the file importing all other files

Reply:
moved that to variables.color